modules/cluster-logging-collector-syslog.adoc

```diff
@@ -96,6 +96,7 @@ parameters within the `<store>` stanza:
 
 .. Specify the *syslog* protocol type:
 +
+[source,terminal]
```

Comment:
I don't think we should use this style for the syslog configuration file.

Reply:
Wht style should be used?